src/legacy/core_plugins/kibana/public/dashboard/index.js

```diff
@@ -60,12 +60,19 @@ uiRoutes
     template: dashboardListingTemplate,
     controller($injector, $location, $scope, Private, config, i18n) {
       const services = Private(SavedObjectRegistryProvider).byLoaderPropertiesName;
+      const kbnUrl = $injector.get('kbnUrl');
       const dashboardConfig = $injector.get('dashboardConfig');
 
       $scope.listingLimit = config.get('savedObjects:listingLimit');
+      $scope.create = () => {
+        kbnUrl.redirect(DashboardConstants.CREATE_NEW_DASHBOARD_URL);
+      };
       $scope.find = (search) => {
         return services.dashboards.find(search, $scope.listingLimit);
       };
+      $scope.edit = ({ id }) => {
+        kbnUrl.redirect(createDashboardEditUrl(id));
```

Comment:
This doesn't really take you to the dashboard edit page, but to the "view" page.

Reply:
This is the previous behavior. I'll discuss with @cchaos and open another task if needed.